src/react-components/ui-root.js

```diff
@@ -963,6 +973,10 @@ class UIRoot extends Component {
     const pendingMessageFieldHeight = textRows * 28 + 20 + "px";
     const hasPush = navigator.serviceWorker && "PushManager" in window;
     const promptForNameAndAvatarBeforeEntry = !this.props.store.state.activity.hasChangedName;
+    const discordBridges = this.discordBridges();
```

Comment:
perhaps this stuff should be factored in a way that will deal nicely with subsequent non-discord bridges?

Reply:
How it is seems good to me. If you wanted to make an abstraction on top of the presence entries that provides some functions for getting out useful information, I'm sure you could do so, but I don't know the best way to do it right now.